packages/modular-scripts/src/check/index.ts

```diff
@@ -10,24 +10,29 @@ const CHECKS = [
 ];
 
 interface Check {
-  check(this: void): Promise<boolean>;
-  fix?(this: void): Promise<void>;
+  fix: boolean;
+  target?: string;
 }
 
-export async function check(fix = false): Promise<void> {
+interface Verification {
+  check(target?: string): Promise<boolean>;
+  fix?(): Promise<void>;
+}
+
+export async function check({ fix = false, target }: Check): Promise<void> {
```

Comment:
For a consistent approach to declaring defaults, the default for `target` can also be defined in the function signature:

```suggestion
export async function check({ fix = false, target = process.cwd() }: Check): Promise<void> {
```